src/main/java/namor/template/calculator/CalculatorFactory.java

```diff
@@ -0,0 +1,17 @@
+package namor.template.calculator;
+
+
+import namor.template.stack.ArrayStack;
+
+public class CalculatorFactory {
+
+    public Calculator getCalculator(String inputString) {
+        String element = String.valueOf(inputString.charAt(inputString.length() - 1));
+
+        if(Sign.isSign(element)) {
+            return new RPNcalculator(new ArrayStack<Double>());
+        }else {
+            return new ClassicCalculator();
+        }
+    }
+}
```

Comment:
I told in description of the task not to write tests, my mistake, I thought you will do everything in Application class. Your `CalculatorConsoleApplication` class is endpoint of application it acts like controller, that's why there is no need for tests.
The factory method is a service of the application it contains application logic and should be tested. So please add unit tests for factory